amgut/templates/register_user.html

```diff
@@ -24,8 +24,12 @@
         address: "required",
         city: "required",
         state: "required",
-        zip: "required",
-        country: "required"
+        zip: {"required": true,
+              "maxlength": 10
```

Comment:
really? I could not find this (looking at the HTML documentation of the schema; I don't have access to the live system at this time)